README.md

```diff
@@ -1 +1,8 @@
-# create202404
\ No newline at end of file
+# create202404
+Some description that doesnt mean anything.
+
+## Section 2
+How to enable something that need to be enabled without the need to enable it.
+
```

Comment:
The sections in the README are not in logical order. It is typical to list sections in a README in a logical or chronological order for better readability.
```suggestion
# create202404
Some description that doesnt mean anything.

## Section 1
Section that should be before #section2 but for some reason is after

## Section 2
How to enable something that need to be enabled without the need to enable it.
```